package.xml

```diff
@@ -46,7 +47,7 @@
     <min>7.4.0</min>
    </php>
    <pearinstaller>
-    <min>1.4.0b1</min>
+    <min>1.10</min>
```

Comment:
Makes sense - I guess even php 7.0 has 1.10 - this was based on package.xml from a project that used to support php 5.x (igbinary)